sdk/eventhub/event-hubs/test/public/receiver.spec.ts

```diff
@@ -15,13 +15,9 @@ import {
   earliestEventPosition,
```

Comment:
Now that this is only testing the `EventHubConsumerClient`, it might be nice to merge the remaining tests into the `test/public/eventHubConsumerClient.spec.ts` file so they're all in one place.

Reply:
@chradek I think if I combine eventHubConsumer and receiver tests the history will be lost :P @richardparkmsft

Reply:
That's not my github alias! :)

I think this is fine - we can't preserve history forever and, more importantly, it's just tests. It was a nice-to-have, not a must. @chradek  is the primary owner of EventHubs and his word is basically law on this.

Reply:
Yeah, I'd rather consolidate the files so I only need to look at one spot to know where to add/modify tests.

The test descriptions should hopefully be good enough where it's clear why a test exists and what it is testing, so I don't mind losing the history (plus it's not really lost if you're willing to go back in time enough!)